src/Workspaces/CSharp/Portable/CodeGeneration/CSharpCodeGenerationService.cs

```diff
@@ -533,6 +537,23 @@ protected override TDeclarationNode AddMembers<TDeclarationNode>(TDeclarationNod
             return Cast<TDeclarationNode>(finalMember);
         }
 
+        private static TDeclarationNode AddStatementsToLocalFunction<TDeclarationNode>(TDeclarationNode destinationMember, IEnumerable<SyntaxNode> statements, LocalFunctionStatementSyntax localFunctionDeclaration) where TDeclarationNode : SyntaxNode
+        {
+            var body = localFunctionDeclaration.Body;
+            if (body == null)
+            {
+                return destinationMember;
+            }
```

Comment:
so what happens in this case.  the user selects "extract local function" and what happens?  nothing?  make sure we have a test for this.

Reply:
In another comment you mentioned that local functions always have to have bodies, so I've gone ahead and removed this check since I don't think it will ever be hit.